src/TraceEvent/TraceEvent.csproj

```diff
@@ -71,6 +71,7 @@
 
   <!-- *** SourceLink Support *** -->
   <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta-63127-02" PrivateAssets="All" />
+  <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
```

Comment:
💡  This should appear after line 70 (so it's not listed as part of SourceLink support)

Reply:
Thanks. I didn't check.